test/e2e/specs/editor/various/post-title.spec.js

```diff
@@ -0,0 +1,303 @@
+				name: 'Add title',
+			} );
+
+			pageUtils.setClipboardData( {
+				plainText: 'hello plain',
+				html: 'hello html',
+			} );
+
+			// focus on the title field
+			await pageTitleField.focus();
+
+			await pageUtils.pressKeys( 'primary+v' );
+
+			await expect( pageTitleField ).toHaveText(
+				'I am <em>emphasis</em> I am <strong>bold</strong> I am <a href="#">anchor</a>'
+			);
+		} );
+
+		test( 'should strip HTML tags from Post Title when pasted text is transformed to blocks', async ( {
+			editor,
+			admin,
+			pageUtils,
+		} ) => {
+			await admin.createNewPost();
+
+			const pageTitleField = editor.canvas.getByRole( 'textbox', {
+				name: 'Add title',
+			} );
+
+			await expect( pageTitleField ).toBeFocused();
+
+			// This HTML will ultimately be parsed into two blocks
+			// The first will have it's `content` attribute stripped of HTML
+			// and used as the Page Title.
+			// The second will be inserted into the post contents and will
+			// retain its HTML.
+			pageUtils.setClipboardData( {
+				html: `
+					<h2>I am heading block title with <strong> HTML tag</strong></h2>
+					<p>And I am the rest of titles with <em>emphasis tag</em>!</p>
+				`,
+			} );
+			await pageUtils.pressKeys( 'primary+v' );
+
+			// Check the HTML elements have been stripped from the first block's
+			// `content` attribute...
+			await expect( pageTitleField ).toHaveText(
+				'I am heading block title with HTML tag'
+			);
+
+			// ...and are not rendered.
+			await expect( pageTitleField.locator( 'css=strong' ) ).toBeHidden();
+
+			// Check the 2nd block ended up in the post contents and did not
+			// have its HTML stripped out.
+			await expect.poll( editor.getBlocks ).toMatchObject( [
+				{
+					name: 'core/paragraph',
+					attributes: {
+						content:
+							'And I am the rest of titles with <em>emphasis tag</em>!',
+					},
+				},
+			] );
+		} );
+
+		test.skip( 'should output HTML tags in plaintext when typed into Post Title field in visual editor mode', async ( {
+			editor,
+			page,
+			admin,
+			pageUtils,
+		} ) => {
+			await admin.createNewPost();
+
+			const pageTitleField = editor.canvas.getByRole( 'textbox', {
+				name: 'Add title',
+			} );
+
+			await expect( pageTitleField ).toBeFocused();
+
+			await page.keyboard.type( 'I am <em>emphasis</em>' );
+
+			await expect( pageTitleField ).toHaveText(
+				'I am <em>emphasis</em>'
+			);
+
+			// Check that the `em` tag was output in plaintext and not rendered.
+			await expect( pageTitleField.locator( 'css=em' ) ).toBeHidden();
+
+			// Switch to code view
+			await pageUtils.pressKeys( 'secondary+M' ); // Emulates CTRL+Shift+Alt + M => toggle code editor
+
+			const codeViewPageTitleField = editor.canvas.getByRole( 'textbox', {
+				name: 'Add title',
+			} );
+
+			// Check that the `em` tag was output in plaintext (HTML entities)
+			await expect( codeViewPageTitleField ).toHaveText(
+				'I am &lt;em&gt;emphasis&lt;/em&gt;'
+			);
```

Comment:
@ellatrix This is failing. Instead of encoding both `<` and `>` for each tag it only encodes the first tag. 

```
I am &lt;em>emphasis&lt;/em>
```

Is that wrong? Shouldn't it encode **both**?

```
I am &lt;em&gt;emphasis&lt;/em&gt;
```

Reply:
That's normal? Encoding closing char is optional

Reply:
✅ Test updated (with comment by way of explanation) and enabled.